link_list.py

```diff
@@ -0,0 +1,80 @@
+class Node(object):
+    """Creating a template for a Node."""
+
+    def __init__(self, data):
+        """providing attribute to Node."""
```

Comment:
Nicely done with having a doc string. Make sure you are specific with your documentation though -- what kind of attributes?